code-challenges.js

```diff
@@ -7,8 +7,24 @@ var collections = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example2 (can be a different order): ["blue", "green", "pink"]
 // Expected output example3 (can be a different order): ["pink", "green"]
 
-
-
+//Declare variable that takes in array
+// const removeFirst = (array) => {
+//     //iterate through array using a for loop
+//     for (let i = array.length - 1; i > 0 ; i--) {
+//         //establish randomizing of array 
+//         var j = Math.floor(Math.random() * (i + 1));
+//         [array[i], array[j]] = [array[j], array[i]]
+//     }
+//     //once randomized remove the first item in the in the array
+//     array.shift()
+//     return array
+// }
+//each time it is called it will remove an item until it just returns an empty array.
+// console.log(removeFirst(collections))
+// console.log(removeFirst(collections))
+// console.log(removeFirst(collections))
+// console.log(removeFirst(collections))
+// console.log(removeFirst(collections))
 
```

Comment:
Interesting approach! Can you do me a favor and leave your code uncommented next time? Thanks!